src/components/extension-landing/extension-landing.scss

```diff
@@ -7,6 +7,11 @@
     padding: 0;
 }
 
+html {
+    // Adjust target position to compensate for the floating top content
+    scroll-padding-top: calc(50px + 5rem); // nav bar + OS chooser
```

Comment:
This one's new to me as well! The `scroll-padding` property pads the scrolling area by the specified amount, so when the browser tries to scroll something into view (like `#troubleshooting`) the browser makes sure that the scrolling target is at least that far away from the edge(s). So, this `scroll-padding-top` means make sure that the scroll target (`#troubleshooting`) is at least `50px + 5rem` -- that's `50px` for the global `scratch-www` nav bar, plus `5rem` for the OS chooser that's on all the extension pages -- away from the top of the window.

I think of it like "pretend the scroll frame is this much smaller than it actually is" -- like padding inside the window frame, but only for the purposes of scrolling and not for the purposes of layout.

I don't think this would be a good idea if we try to scroll to anchors near the top of the page because of the dynamic floatiness of the OS chooser. If we ever have to deal with that, I think this will get much more complicated.